egs/wsj/s5/steps/nnet3/get_egs_targets.sh

```diff
@@ -393,10 +484,26 @@ if [ $stage -le 5 ]; then
     $cmd --max-jobs-run $nj JOB=1:$num_archives_intermediate $dir/log/shuffle.JOB.log \
       nnet3-shuffle-egs --srand=\$[JOB+$srand] "ark:cat $egs_list|" ark:- \| \
       nnet3-copy-egs ark:- $output_archives || exit 1;
+
+    if $generate_egs_scp; then
+      #concatenate egs.JOB.scp in single egs.scp
+      rm -rf $dir/egs.scp
+      for j in $(seq $num_archives_intermediate); do
+        for y in $(seq $num_archives_intermediate); do
+          cat $dir/egs.$j.$y.scp || exit 1;
+        done
+      done > $dir/egs.scp || exit 1;
+      for f in $dir/egs.*.*.scp; do rm $f; done
+    fi
   fi
+fi
 
+if [ $frame_subsampling_factor -ne 1 ]; then
+  echo $frame_subsampling_factor > $dir/frame_subsampling_factor
```

Comment:
chain/get_egs.sh does not put anything in egs/info. Instead train.py does. So it is really a bug, which was never discovered because the frame-subsampling-factor is always 3 and no one tried running get_egs.sh outside of train.py with a different frame_subsampling_factor from what is passed to train.py. 
Also I expected train_dnn.py etc. to copy frame_subsampling_factor from egs directory in a way it does with cmvn_opts. But this is not the case.